templates/items/tool.hbs

```diff
@@ -84,7 +84,7 @@
             <div class="form-group">
                 <label>{{ localize "DND5E.DefaultAbilityCheck" }}</label>
                 <select name="system.ability">
-                    {{selectOptions config.abilities selected=system.ability}}
+                    {{selectOptions config.abilities selected=system.ability blank=(localize "DND5E.Default")}}
```

Comment:
Seems like there should also be a default or automatic option for proficiency.

Reply:
That will be coming in a followup PR. I intend to do an 'automatic proficiencies' PR across weapons, armour, and tools, but it's too big for one PR. For now, the proficiency on the tool is just ignored.